### ISSUE TITLE
Error Message: local variable 'latestRelease' referenced before assignment

### ISSUE THREAD

**Issue by sbeaudet** [ThioJoe/YT-Spammer-Purge#993]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Did Check for Updates on newest beta 2.17.0-Beta6 and got error X-2

### Release version

2.17.0-Beta6

### Steps to reproduce

1. Update to newest beta
2. Check for Updates

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1715, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1692, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 930, in primaryInstance
  File "Scripts\files.py", line 87, in check_lists_update
UnboundLocalError: local variable 'latestRelease' referenced before assignment
------------------------------------------------
Error Message: local variable 'latestRelease' referenced before assignment

Error - Code: X-2 occurred.
```

**Comment by Firecul** [comment]
While not particularly useful for you, it seems to be working on the Python version at least.
![image](https://user-images.githubusercontent.com/34950371/208003917-50a66e10-5314-4db3-8d9e-4d3c1e1b8d92.png)

**Comment by ThioJoe** [comment]
Hm odd, the exe seems to work fine for me when checking.

Also looking at the code, it's hard to tell what could possibly have caused that error. Every condition seems to be accounted for. The only possibility I see is if it was specifically an "OSError" that was not accounted for. (The only one I have accounted for is (WinError 10013). 

In the latest commit I've added code to handle it and at least print the error from now on. So it will be in the next release whenever that is.

Unfortunately I'm not sure what the problem was, but it was apparently system related. Perhaps a firewall or local connection issue? Does it still happen now?
